### ISSUE TITLE
Add per-file coverage tresholds

### ISSUE THREAD

**Issue by Plackier** [mattlewis92/karma-coverage-istanbul-reporter#12]
Hello,
It's an enhancement proposition - I think it would be beneficial to add **per-file** code coverage tresholds support, like in original karma-coverage.

**Comment by mattlewis92** [comment]
This should be possible, quickly looking at the source it seems I can get per file coverage like so:
```
const summary = remappedCoverageMap.getCoverageSummary();
remappedCoverageMap.files().forEach(function (key) {
  console.log(key, remappedCoverageMap.fileCoverageFor(key).toSummary());
});
```

**Comment by mattlewis92** [comment]
Released as 1.2.0 🎉
